### PR TITLE
fix inline css in emails

### PR DESCRIPTION
**Pull request by robguthrie**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -74,7 +57,7 @@ gem 'simple-rss'
 gem 'snorlax', '~> 0.1.3'
 gem 'custom_counter_cache'
 gem 'safe_shell'
-gem 'premailer'
+gem 'premailer-rails'
```
Comment (robguthrie) [inline Gemfile]: problem 1: we were using the wrong gem.

Comment (robguthrie) [inline Gemfile]: problem 2, there was an email.png which was being referenced rather than email.css

File: config/application.rb [hunk 2]
```diff
@@ -6,6 +6,17 @@
 
 require_relative '../lib/version'
 
+def lmo_asset_host
```
Comment (gdpelican) [inline config/application.rb]: I have found myself wanting access to this in other parts of the application as well

(namely the emojifier, to ensure that we're displaying images from our server in emails)
I guess with this I could then use `Rails.application.config.action_mailer.asset_host`?

Comment (robguthrie) [inline config/application.rb]: Yep!

**Comment by robguthrie** [comment]
ok to merge @gdpelican ?

**Comment by gdpelican** [comment]
Green bean!

**Pull request merged**